react/src/components/Table/Table.tsx

```diff
@@ -556,7 +553,28 @@ const Table: React.FC<TableProps> = ({ variantData }) => {
                                                                 <Flex
                                                                     alignItems="center"
                                                                     justifyContent="center"
+                                                                    nowrap
                                                                 >
+                                                                    {/* 
+                                                                        Filter icon for filtering individual columns
+                                                                    */}
```

Comment:
This component may be getting too complicated if the indentation is going so far that the formatter is doing this to comments.

Reply:
Yeah.... I'll make a separate issue for thinking about how to make `Table.tsx` less complicated.